src/StatusQ/Buttons/StatusTextWithIconButton.qml

```diff
@@ -0,0 +1,104 @@
+import QtQuick 2.14
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+
+Rectangle {
+    id: statusTextWithIconButton
+
+    property alias text: label.text
+    property alias font: label.font
+
+
+    property color iconNormalColor: "transparent"
+    property color iconDisabledColor: "transparent"
+
+    property color backgroundNormalColor: "transparent"
+    property color backgroundHoverColor: "transparent"
+    property color backgroundDisabledColor: "transparent"
```

Comment:
Just like I've mentioned in the other comment: We don't want consumers of this component to be in charge of assigning all the colors.

Reply:
Yes but its can be used as base component. If in future we never want to create some red button with icon - no problem. But i don't think so.

Reply:
If  we want to treat it as a base component, then this PR should also introduce the more concrete equivalents.

But since we have a `Theme` system in place for that, I don't think we have to start with a `transparent`-all-the-way component.